src/Main.java

```diff
@@ -7,77 +7,119 @@
 
 public class Main {
 
-    static void drawLine(BufferedImage img, boolean w1, boolean w2, boolean w3, boolean h1, boolean h2, boolean h3, boolean h4) {
+    public static void main(String[] args) throws IOException {
+        BufferedImage img = new BufferedImage(500, 200, BufferedImage.TYPE_INT_RGB);
         Graphics2D g2d = img.createGraphics();

```

Comment:
Would be more clean if you extract 
```
Graphics2D g2d = img.createGraphics();
g2d.setColor(Color.WHITE);
g2d.fillRect(0, 0, img.getWidth(), img.getHeight());
g2d.dispose();
```
to separate method, let's say drawBackground, or clearToColor, or fillWithWhite, or sth like that

Reply:
Done, separate to `drawBackground`